src/main/java/com/github/dockerjava/api/model/HostConfig.java

```diff
@@ -27,6 +27,9 @@
 	@JsonProperty("Privileged")
 	private boolean privileged;
 
+	@JsonProperty("ReadonlyRootfs")
+	private boolean readonlyRootfs;
```

Comment:
Good point. I think we should change all json mapped `boolean` fields into `Boolean` in a separate refactoring.